01/app.js

```diff
@@ -13,10 +13,11 @@ function addUser(e) {
     }
     if (lastName.length === 0) {
         alert('Last Name is required!');
+    } else {
```

Comment:
Źle zrobiony warunek. Jak będzie nazwisko, a nie będzie imienia to kolejny element zostanie dodany.

Reply:
Zmienione